recipes/julia/build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-make
-env PREFIX=$PREFIX prefix=$PREFIX make install
+mv * "$PREFIX"
+mv "$PREFIX/LICENSE.md" .
```

Comment:
I see. This is because of the `license_file` statement in the meta.yaml. Seems to be necessary, because julia specifies some exceptions to MIT license.